.travis.yml

```diff
@@ -6,12 +6,20 @@ addons:
       - xvfb
 
 env: 
-  - FIREFOX_VERSION="latest"
-  - FIREFOX_VERSION="60.0.1"
+  global: 
+    - # Environment variable for WebDriverManager see https://github.com/bonigarcia/webdrivermanager#known-issues
+    - WDM_GITHUBTOKENNAME="for-travis-ci"
+    - secure: "W9cR3N+NP/d8IJ/WdGuvdcOo31Agi4Qr0fH+Hd1MKrJHsH1d8JOStlQsec4Ih7aYlq2c1XRj1XHg7xrc9lH3JJxh7JkXr8XZIe+Fd1gdOKD23vI+4e+HfCeYVdv56Ia7fTBDCC+5L23x8F4zuyLRlfbPXUX9CrltKDtEfKq/XYKjJcESbOJSJiAsSNjFgd0AECvLLEsBZxh2KseJytAxoNT/ScYmoE+30N6o8Gke26IhiNEhCP43NUD5HzcfdDmjRaBjQPXTN1AjfjHxXmcqTxeBiqfSatKg7BplSm8zvCR2MnaJHQJCwD7hUVfrVGa8mNzVycuu2FimBpg9wGOFEqlOZ2tGx3Xy3siLVCfbZc6LykBeHdoZjhjsTF+NqYxmsFAWvfPIJudViB9qk5uODTMvgJdk1nYStuKQLmGjzDoCtnRoV3RWExhMZsSW2Io1jr/uLHKDCqWidYqyoFvIy++svwk6vte+IwQ3TTSrrff6RdddIs/oVeSI+aItffvXKbpEJHkRy5ZP/jh9fJS2UeVitSON09XEx4h2lVCgzDqNNN5wV78CJGhCY3GlrEjGOmnK5kpASoKEKbTh4ebNkkIBZD6npF7Kqz8HApafHhWmvKIxcc8HWz26qJp75XIX8j36Ix+2IBPoZ4zbhC6tVh8FexqH4pkArJ7nYmJpDHg="
+    
+  matrix: 
+    - FIREFOX_VERSION="latest"
+    - FIREFOX_VERSION="60.0.1"
  
 before_install:
   # Install firefox for testing.
   - echo $FIREFOX_VERSION
+  - echo $WDM_GITHUBTOKENNAME
+  - echo $WDM_GITHUBTOKENSECRET
```

Comment:
even though travis does not echo this (thank god) you can as well remove this line, right?

Reply:
Jupp, this is the account of srvte. But i created a new Personal Access Token like it is mentioned in the documentation of Boni Garcia. And yes i deleted the two lines ;O)